src/components/MultiChoice/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import classNames from 'classnames';
+
+import Checkbox from 'components/Checkbox';
+import { Answer } from 'types/answer';
+
+export const multiChoiceDataTestIds = {
+  base: 'multi-choice__base',
+};
+
+interface MultiChoiceProps {
+  items: Answer[];
+  isPickOne: boolean;
+  onValuesChanged: (answers: Answer[]) => void;
+}
+
+const MultiChoice = ({ items, isPickOne, onValuesChanged }: MultiChoiceProps): JSX.Element => {
+  const [selectedValues, setSelectedValues] = useState<Answer[]>([]);
+
+  const isSelected = (answer: Answer): boolean => {
+    return selectedValues.find((item) => item.id === answer.id) !== undefined;
```

Comment:
```suggestion
    return !!selectedValues.find((item) => item.id === answer.id);
```

Reply:
@nvminhtue wow, thanks for this tip 🤩 TIL